pulp_smash/pulp3/constants.py

```diff
@@ -16,7 +16,7 @@
 
 BASE_PUBLISHER_PATH = urljoin(BASE_PATH, "publishers/")
 
-CONTENT_GUARDS_PATH = urljoin(BASE_PATH, "contentguards/certguard/certguard/")
+CONTENT_GUARDS_BASE_PATH = urljoin(BASE_PATH, "contentguards/")
```

Comment:
```suggestion
BASE_CONTENT_GUARDS_PATH = urljoin(BASE_PATH, "contentguards/")
```

Reply:
+1 to this, this is more consistent with what is here.